python/Dockerfile

```diff
@@ -1,5 +1,18 @@
-FROM python:3.5
-VOLUME /usr/src/scripts
-WORKDIR /usr/src/scripts
+FROM continuumio/miniconda3
 
-ENTRYPOINT ["python"]
+# Set up code directory
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY environment.yml .
+
+# Install dependencies
+RUN conda env create -f environment.yml
```

Comment:
I know nothing about python.  Is conda the current go to?

Reply:
The benefit of conda is that it is both a package and version manager. So people can use this same image and switch to whatever python version they want (typically one o f 2.7, 3.4, 3.5, 3.6); and, they can also manage the virtual environment with dependencies. 

Conda is the most common approach on Windows; it's less common on mac simply because Mac comes with python already on it. It's not quite the current go to I think but it is widely used. Mostly it's what one of our volunteers chose to switch to when he needed to add automatic virtual environment creation, so we have it working.

Reply:
Sounds reasonable to me.